src/frontend/templates/pages/games_list_by_user.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+
+{% block title %}
+{% if kind == "training" %}
+{% blocktrans with user_name=user.username run_name=run.name %} KataGo - Training Games from {{user_name}} for {{run_name}} {% endblocktrans %}
+{% else %}
+{% blocktrans with user_name=user.username run_name=run.name %} KataGo - Rating Games from {{user_name}} for {{run_name}} {% endblocktrans %}
+{% endif %}
+{% endblock %}
+
+{% block content %}
+
+{% if kind == "training" %}
+<h1 class="title">{% blocktrans with user_name=user.username run_name=run.name %} Training Games from {{user_name}} for {{run_name}} {% endblocktrans %}</h1>
+{% else %}
+<h1 class="title">{% blocktrans with user_name=user.username run_name=run.name %} Rating Games from {{user_name}} for {{run_name}} {% endblocktrans %}</h1>
+{% endif %}
+
+{% include 'pagination.html' with other_get_params='' page_obj=page_obj %}
+
+<table class="table">
+  <tr>
+    <th> {% trans "Game ID" %} </th>
+    <th> {% trans "Uploaded At" %} </th>
+    {% if kind == "rating" %}
+    <th> {% trans "Black" %} </th>
+    <th> {% trans "White" %} </th>
+    {% else %}
+    <th> {% trans "Network" %} </th>
+    {% endif %}
```

Comment:
That feeels weird for me.

ideally I think we could have two tables, one for "rating", one for "training games" and just "if kind" one in the vue to include the good table rather than "if kind" x times in the table.

That would increase cohesion and reduce complexity.

Reply:
Thanks, I will consider it. Right now it is the *only* thing in the table that varies between rating and training - is it really good style to duplicate all the code for 11 other identical columns to save on 1 if statement?